packages/expo-video/src/VideoPlayer.tsx

```diff
@@ -1,8 +1,14 @@
 import { useReleasingSharedObject } from 'expo-modules-core';
+import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
 
 import NativeVideoModule from './NativeVideoModule';
 import type { VideoPlayer, VideoSource } from './VideoPlayer.types';
 
+const replace = NativeVideoModule.VideoPlayer.prototype.replace;
+NativeVideoModule.VideoPlayer.prototype.replace = function (source: VideoSource) {
+  return replace.call(this, parseSource(source));
+};
```

Comment:
Could you add a comment that this is only a temporary solution until we figure out a better way to override prototypes without having to load the entire module on startup?